tests/test-component.php

```diff
@@ -63,90 +66,574 @@ public static function create_test_components() {
 	 */
 	public function get_component( $name ) {
 		if ( isset( self::$components[ $name ] ) ) {
-			return self::$components[ $name ];
+			return clone self::$components[ $name ];
```

Comment:
might be worth a note here, I guess this lets each test run on a fresh component?

Reply:
Yeah. It's cheaper than reloading the components somehow.